README.md

```diff
@@ -1 +1,37 @@
-# jest-fixed-jsdom
+<h1 align="center">jest-fixed-jsdom</h1>
+<p align="center">This library helps re-attach missing NodeJS globals definitions in JSDOM that Jest strips out</p>
+
+## What is this for?
+
+Jest strips out a number of NodeJS globals that are used in tests and libraries involving JSDOM, such as structuredClone, ReadableStream, and so on. This library patches these globals back in - there is no polyfilling or mocking involved, it simply re-attaches the missing globals to the JSDOM environment. If you've ever come across errors such as `ReferenceError: ReadableStream is not defined` or `ReferenceError: structuredClone is not defined`, this library is for you. If you were previously using (undici)[https://www.npmjs.com/package/undici] purely to solve this, you will no longer need it. 
+
+## Installation
+
+If using yarn:
+
+`yarn add jest-fixed-jsdom --dev`
+
+
+If using npm:
+
+`npm install jest-fixed-jsdom -D`
+
+If using pnpm:
+
+`pnpm add jest-fixed-jsdom -D`
+
+## Configuring your jest environment to use jest-fixed-jsdom
+
+You will need to add/modify two properties in your jest configuration to use jest-fixed-jsdom. 
+
+```json
+{
+  "testEnvironment": "jest-fixed-jsdom",
+  "testEnvironmentOptions": {
+    "customExportConditions": [""],
```

Comment:
I think we can skip this part as you don't have to change the export conditions for this custom environment to work. For now, let's omit it. We may implement a custom `resolver` like you've proposed but that needs more thought.

Reply:
That's true - I was thinking of the end-user using msw, rather than the more generic case. Perhaps a custom resolver solution would be worth putting into the `msw` docs somewhere instead?

Reply:
Yeah, let's explore the resolver option separately from this change. 

> Perhaps when (I'm assuming you'll want to do this) this library is added to the msw docs, a note about this can go alongside it?

This will definitely be mentioned as you said in the MSW docs.